pass-data-client/src/main/java/org/eclipse/pass/support/client/model/AggregatedDepositStatus.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Johns Hopkins University
+ * Copyright 2023 Johns Hopkins University
```

Comment:
Given the recent decision to change the copyright statement structure in our project, and that the year update is the only change in the pass-data-client module files, I'd revert this change just to limit the number of files changed in this PR.  Also, I think I remember hearing that the year doesn't have to be updated even with the new copyright statement structure.

Reply:
Done.